python/BUILDING.md

```diff
@@ -2,8 +2,8 @@
 
 ## Building nGraph Python Wheels
 
-If you want to try a newer version of nGraph's Python API than is available from
-PyPI, you can build your own latest version from the source code. This 
+If you want to try a newer version of nGraph's Python API than is available 
+from PyPI, you can build your own latest version from the source code. This 
```

Comment:
(while we're here) maybe replace `your own latest version` with `the latest version`?

Reply:
Done